exploring.rst

```diff
@@ -0,0 +1,68 @@
+.. _exploring:
+
+Exploring the CPython's Internals
+=================================
+
+This is a quick guide for who want to know about CPython's internals,
```

Comment:
Perhaps:

This guide gives an overview of CPython's code structure. It serves as a summary of file locations for modules and builtins.